package.json

```diff
@@ -28,11 +28,11 @@
   },
   "dependencies": {
     "passport": "^0.3.2",
-    "underscore": "^1.8.2",
-    "eslint": "^2.5.3",
-    "eslint-config-loopback": "^1.0.0"
+    "underscore": "^1.8.2"
   },
   "devDependencies": {
+    "eslint": "^2.5.3",
```

Comment:
2.13.1 was just the latest 2.x when the post was written. There is no significance to the specific version - they just point people at their latest release like every other dev does (including us!)